package.json

```diff
@@ -51,12 +51,5 @@
     "tape": "^4.5.1"
   },
   "dependencies": {
-    "babel-plugin-transform-es2015-destructuring": "^6.9.0",
```

Comment:
:+1: chur. can we use `es2040` instead? we also need to update the `browserify.transform` field to match.